nixos/modules/services/games/asf.nix

```diff
@@ -158,6 +158,7 @@ in
 
     users = {
       users.asf = {
+        createHome = true;
```

Comment:
There was a lot of discussion about this in the initial PR and it was determined to be better to use StateDirectory

See here: https://github.com/NixOS/nixpkgs/pull/148541#discussion_r762430559

Reply:
Oh, yeah that makes a lot of sense. I also reverted one commit from yesterday.